module/Company/src/Company/Model/Company.php

```diff
@@ -360,7 +360,7 @@ public function getNumberOfActiveJobs($category = null)
     public function getNumberOfExpiredPackages()
     {
         return count(array_filter($this->getPackages()->toArray(), function ($package) {
-            return $package->isExpired(new DateTime);
+            return $package->isExpired(new DateTime());
```

Comment:
Does this change have the same behaviour? The old code looked like a bug to make anyway

Reply:
There is no difference in behaviour between the two versions. Hence it is a choice for the developer whether to use them or not.

However, looking at our codebase we use parentheses for our own models. Furthermore, if we want to keep adhering to PSR, we must take into account that PSR-12 dictates that parentheses must be present when instantiating a new class. Additionally, all official PHP sources also use parentheses (iirc).

Reply:
With parentheses makes way more sense to me anyway